resources/packs/gcp/gcp.lr

```diff
@@ -1358,3 +1360,73 @@ private gcp.project.loggingservice.bucket.indexConfig {
   // Type of data in this index
   type string
 }
+
+// GCP Cloud function
+private gcp.project.cloudFunction {
+  // Project ID
+  projectId string
+  // Cloud function name
+  name string
+  // Cloud function description
+  description string
+  // Location of the archive with the function's source code
+  sourceArchiveUrl string
+  // Repository reference for the function's source code
+  sourceRepository dict
+  // Location of the upload with the function's source code
+  sourceUploadUrl string
+  // HTTPS endpoint of source that can triggered via URL
+  httpsTrigger dict
+  // Source that fires events in response to a condition in another service
+  eventTrigger dict
+  // Status of the function deployment
+  status string
+  // Name of the function (as defined in source code) that will be executed
```

Comment:
```suggestion
  // Name of the function (as defined in source code) that is executed
```

Whenever possible, I try to use present tense. But please let me know if it really breaks the meaning!